ibi/coreos-installer/Makefile

```diff
@@ -76,3 +82,32 @@ ostree-restore: credentials
 	scp certs.tgz ostree-restore.sh $(TMPDIR)/*-secret.json core@$(VM_NAME):/var/tmp
 	ssh core@$(VM_NAME) sudo /var/tmp/ostree-restore.sh $(SEED_IMAGE)
 
+.PHONY: certs
+certs:
+	@echo "Generating new certificates"
+	rm -rf ./certs ./kubeconfig
+	./scripts/generate_certs.sh
+
+.PHONY: $(CLUSTER_CONFIG_PATH)
+$(CLUSTER_CONFIG_PATH):
+	rm -rf $@
+	VM_NAME=$(VM_NAME) \
+	CLUSTER_NAME=$(CLUSTER_NAME) \
+	DOMAIN=$(DOMAIN) \
+	HOSTNAME=$(HOSTNAME) \
+	scripts/generate_config.sh $(CLUSTER_CONFIG_TEMPLATE) $@
```

Comment:
We can use envsubst to perform the substitutions, as we do in
https://github.com/javipolo/redhat-pocs/blob/d2cac74a8e7a1b7cb2c93f9fd629aa8501cc0599/ibi/coreos-installer/Makefile#L30-L32

That way we can avoid having a separate script for this, and we'll also use the same way of handling templated files in all the project

Reply:
the reason I took it out of the makefile into another script is because I can't seem to get the VM IP into a variable in the makefile.

Reply:
NM, sort it out

Reply:
```
ip: MASTER_IP=$(shell virsh domifaddr ${VM_NAME} | awk -F " " '/ipv4/{print $$4}' | cut -d'/' -f1)
ip:
        @echo ${MASTER_IP}
```

This works for me

it's tricky having to "define" variables this way, but I did not find a different way to define target-specific variables in makefiles